modules/ROOT/pages/local_development.adoc

```diff
@@ -0,0 +1,139 @@
+include::{partialsdir}/attributes.adoc[]
+
+= Local Development
+
+To facilitate the development of mobile services on a users 
+local machine or trialling {product-name} without requiring 
+extra hardware, it is possible to bring up {product-name} on 
+your local machine. This guide takes you through the process 
+of setting up {product-name} locally.
+
+== Requirements
+
+=== Ansible
+
+Ansible version 2.3 or later is a requirement of running the 
+installation scripts on your local machine, follow the 
+installation steps
+link:https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html[here].
+
+=== Node.js
+
+Node.js version 8 or later and the bundled package manager 'npm' 
+are a requirement for certain scripts in the installation process, 
+install these with the instructions:
+link:https://nodejs.org/en/download/package-manager/[here].
+
+=== OpenShift Client Tools
+
+The OpenShift client tools are a requirement and must be 3.9 or greater,
+they can be downloaded from 
+link:https://www.openshift.org/download.html[here].
+
+=== Mobile-core Installer
+
+We configure the local development installation of {product-name}
+using ansible scripts in our link:https://github.com/aerogear/mobile-core[mobile-core] repo.
+
+Clone this repo to your local machine and check out the 1.0.0-alpha tag using:
```

Comment:
will it still be `1.0.0-alpha` after the community release or will everything be retagged with `1.0.0`?

Reply:
Best to use variable here :D